components/apimgt/org.wso2.carbon.apimgt.gateway/pom.xml

```diff
@@ -231,11 +231,15 @@
             <groupId>org.wso2.carbon.mediation</groupId>
```

Comment:
Should we validate th signature of the token via JWKS endpoint?

Reply:
Added with https://github.com/wso2/carbon-apimgt/pull/8198